soc/arm/nxp_lpc/lpc54xxx/Kconfig.soc

```diff
@@ -10,6 +10,7 @@ choice
 config SOC_LPC54114_M4
 	bool "SOC_LPC54114_M4"
 	select CPU_CORTEX_M4
+	select CPU_CORTEX_M_HAS_DWT
```

Comment:
Add this for LPC55xxx as well, I can confirm from DWT is available.
Reference manual ch.52 (Debug subsystem) lists "data watch-points" and SDK examples.

Reply:
Done